stdlib/testing/usage/api.flux

```diff
@@ -0,0 +1,1688 @@
+
+#group,false,false,true,true,false,false,true,true,true,true,true,true
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,long,string,string,string,string,string,string
+#default,_result,,,,,,,,,,,
+,result,table,_start,_stop,_time,_value,_field,_measurement,endpoint,hostname,org_id,status
+,,34,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T13:54:17.541860461Z,135644,req_bytes,http_request,/api/v2/write,gateway-77ff9ccb7d-99mcc,043e0780ee2b1000,204
+,,34,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T13:58:36.538494592Z,135653,req_bytes,http_request,/api/v2/write,gateway-77ff9ccb7d-99mcc,043e0780ee2b1000,204
+,,34,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T13:59:27.740160451Z,134048,req_bytes,http_request,/api/v2/write,gateway-77ff9ccb7d-99mcc,043e0780ee2b1000,204
+
+#group,false,false,true,true,false,false,true,true,true,true,true,true
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,long,string,string,string,string,string,string
+#default,_result,,,,,,,,,,,
+,result,table,_start,_stop,_time,_value,_field,_measurement,endpoint,hostname,org_id,status
+,,35,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T11:35:21.030442937Z,90,req_bytes,http_request,/api/v2/write,gateway-77ff9ccb7d-6pqwb,03d333bccff17000,204
+,,35,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:40:21.02694397Z,90,req_bytes,http_request,/api/v2/write,gateway-77ff9ccb7d-6pqwb,03d333bccff17000,204
+,,35,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T12:50:21.22836104Z,90,req_bytes,http_request,/api/v2/write,gateway-77ff9ccb7d-6pqwb,03d333bccff17000,204
+,,35,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T13:35:21.229099874Z,88,req_bytes,http_request,/api/v2/write,gateway-77ff9ccb7d-6pqwb,03d333bccff17000,204
+
+#group,false,false,true,true,false,false,true,true,true,true,true,true
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,long,string,string,string,string,string,string
+#default,_result,,,,,,,,,,,
+,result,table,_start,_stop,_time,_value,_field,_measurement,endpoint,hostname,org_id,status
+,,36,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T11:00:01.017450939Z,379321,req_bytes,http_request,/api/v2/write,gateway-77ff9ccb7d-vcbpg,03cbe13cce931000,204
+,,36,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T11:05:00.600950432Z,327507,req_bytes,http_request,/api/v2/write,gateway-77ff9ccb7d-vcbpg,03cbe13cce931000,204
+,,36,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T11:10:00.609220694Z,372594,req_bytes,http_request,/api/v2/write,gateway-77ff9ccb7d-vcbpg,03cbe13cce931000,204
+,,36,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08-01T11:10:01.09095546Z,287155,req_bytes,http_request,/api/v2/write,gateway-77ff9ccb7d-vcbpg,03cbe13cce931000,204
+,,36,2019-08-01T11:00:00Z,2019-08-01T14:00:00Z,2019-08
```

Comment:
Is this query right?  If it's not filtering on `_field` isn't it getting too many points for the count?